crates/web/frontend/next.config.mjs

```diff
@@ -1,4 +1,14 @@
+const apiEndpoint = process.env.API_ENDOINT || "http://localhost:3001";
```

Comment:
```suggestion
const apiEndpoint = process.env.API_ENDPOINT || "http://localhost:3001";
```

Reply:
remember to modify the frontend's Dockerfile to allow this env var

See [this](https://docs.docker.com/reference/dockerfile/#env).

the docker compose files that use the frontend should be updated too (for example https://github.com/jorgehermo9/gq/blob/17e83b56d52f7843203ac44757f3863262ca4f0f/docker-compose.yml#L5)